### PR TITLE
Fix of the login form after Flask-Security and WTForms upgrade

### PR DESCRIPTION
**Pull request by quaxsze**
Related to https://github.com/etalab/data.gouv.fr/issues/1053

### CODE REVIEW

File: udata_front/theme/gouvfr/templates/macros/forms.html [hunk 1]
```diff
@@ -7,13 +7,14 @@
 {% elif f.type == 'SubmitField' %}
 <button type="submit" class="fr-btn">{{ submit }}</button>
 {% elif f.type == 'BooleanField' %}
-<div class="fr-checkbox-group">
-    {{ f() }}
-    <label class="fr-label" for="{{ f.id }}">
-        {{ f.label.text }}
-    </label>
-</div>
+{{ render_boolean_field(f) }}
```
Comment (maudetes) [inline udata_front/theme/gouvfr/templates/macros/forms.html:10]: Did you take this update from Flask Security forms?

**Review by maudetes — COMMENTED** [review]
Don't forget the changelog ;)

I think the security override init may not work anymore. `captcha_code` attribute is missing when trying to display http://dev.local:7000/fr/register

File: udata_front/theme/gouvfr/templates/macros/forms.html (hunk 1 above)
Comment (quaxsze) [reply udata_front/theme/gouvfr/templates/macros/forms.html:10]: no I made a func for it:
```
{% macro render_boolean_field(f) %}
<div class="fr-checkbox-group">
    {{ f() }}
    <label class="fr-label" for="{{ f.id }}">
        {{ f.label.text }}
    </label>
</div>
{% endmacro %}
```
I did this so i can reorganize the login form after the update

**Review by maudetes — APPROVED** [review]
Seems like we're clear! :clap: 
Don't forget the changelog!

**Pull request merged**